src/com/facebook/buck/cli/QuickstartCommand.java

```diff
@@ -71,6 +77,19 @@ public String getDestDir() {
   }
 
   public String getAndroidSdkDir() {
+    if (androidSdkDir == null) {
+      ProjectFilesystem projectFilesystem = new ProjectFilesystem(Paths.get("."));
+
+      AndroidDirectoryResolver androidDirectoryResolver =
+          new DefaultAndroidDirectoryResolver(projectFilesystem,
+              Optional.<String>absent(),
+              new DefaultPropertyFinder(projectFilesystem, ImmutableMap.copyOf(System.getenv())));
```

Comment:
No, the daemon's `System.getenv()` will stay the same even if the client's environment changes. In this case I think you can use `params.getRepository().getAndroidDirectoryResolver()` to get the resolver and avoid creating a new instance.